class-plugin-autoupdate-filter.php

```diff
@@ -127,9 +130,49 @@ public function plugin_autoupdate_filter_custom_debug_email( $email, $failures,
 	 * @return string Customized HTML for automatic update settings.
 	 */
 	public function plugin_autoupdate_filter_custom_setting_html( $html, $plugin_file, $plugin_data ) {
+
+		// check if updates are explicitly blocked for this plugin
+		if ( function_exists( 'disable_autoupdate_specific_plugins' ) ) {
+
+			// create a fake object to feed to disable_autoupdate_specific_plugins
+			$plugin_obj                    = new stdClass();
+			$plugin_obj->slug              = dirname( $plugin_file );
+			$plugin_allowed_to_update_bool = disable_autoupdate_specific_plugins( true, $plugin_obj );
+
+			if ( false === $plugin_allowed_to_update_bool ) {
```

Comment:
If I'm understanding this right, this adds it to the notice next to the plugin itself, but won't put the alert at the top of the admin page where it will be more visible. The message next to the plugin doesn't appear in the path the issue was meant to address: a partner doing bulk updates from the **Dashboard > Updates** tab.

Reply:
Great job making sure this actually solves the problem. Fixed that by adding in `admin_notices` for each plugin that is explicitly prevented from autoupdating.
![SCR-20230908-nmfn](https://github.com/a8cteam51/plugin-autoupdate-filter/assets/2067992/7cf3bf26-de1d-498d-a991-aef3bb99a2dc)